packages/scripts/scripts/test-e2e.js

```diff
@@ -42,4 +42,9 @@ const runInBand = ! hasRunInBand ?
 	[ '--runInBand' ] :
 	[];
 
+if ( hasCliArg( 'interactive' ) ) {
+	process.env.PUPPETEER_HEADLESS = false;
+	process.env.PUPPETEER_SLOWMO = 80;
```

Comment:
80 is arbitrary and may produce a different experience depending on the machine where it is executed. I'm fine with making it default to such value, but there should be also an option to change this value, too.

Reply:
Yes, you are right, magic number there. I'll add a custom command for specifying the speed :)